src/utils/bigint.ts

```diff
@@ -123,26 +122,6 @@ export function parseJsonBigInt<T>(text: string, schema: ZodSchema<T>): T {
   return parseSchema(text, jsonSchema);
 }
 
-/**
- * A custom JSON encoding for LevelDB with support for `bigint` properties, powered by Zod schemas.
- *
- * If the resulting JSON contains large integers that would lose precision with the `number` type, they're parsed as `bigint`s.
- * This means that `z.bigint()` properties would fail for small integers, as they would be parsed as `number`s.
- * To mitigate this, use the `bigIntCoercibleSchema` utility, which will coerce the property to a `bigint` output.
- */
-export function jsonBigIntEncoding<T>(schema: ZodSchema<T>): IEncoding<T, string, T> {
```

Comment:
Why are you removing this? Is it used only with leveldb?

Reply:
Yes, IEncoding is a level-transcoder interface.

Level enables a JSON encoding to save whole objects under a key encoded as a JSON string.
This method creates a custom encoder that allows saving objects with bigint values.
The schema was a way to validate the data being decoded.